src/style.css

```diff
@@ -2,20 +2,22 @@ body {
   margin: 0;
 }
 
-#red {
-  background-color: red;
+.block {
   height: 100px;
   width: 100%;
 }
 
-#green {
+.green {
   background-color: green;
-  height: 100px;
-  width: 100%;
+
 }
 
-#blue {
+.blue {
```

Comment:
The theme of the site may be changed and binding class names to colors is bad practice. You can think of these blocks as content, wrapper, and inner wrapper. Example: outer, inner, wrapper, content, etc.